libraries/AP_InertialSensor/AP_InertialSensor_MPU9250.cpp

```diff
@@ -57,6 +59,7 @@ extern const AP_HAL::HAL &hal;
 #       define MPUREG_SMPLRT_100HZ                              0x09
 #       define MPUREG_SMPLRT_50HZ                               0x13
 #define MPUREG_CONFIG                                           0x1A
+#define MPUREG_CONFIG_FIFO_MODE_STOP                    0x40
```

Comment:
its deliberately mis-aligned, as it isn't a register definition, its a bit within the register